packages/mobile/src/verify/VerificationLoadingScreen.tsx

```diff
@@ -0,0 +1,58 @@
+import colors from '@celo/react-components/styles/colors'
+import * as React from 'react'
+import { withNamespaces, WithNamespaces } from 'react-i18next'
+import { ScrollView, StyleSheet } from 'react-native'
+import SafeAreaView from 'react-native-safe-area-view'
+import componentWithAnalytics from 'src/analytics/wrapper'
+import Carousel, { CarouselItem } from 'src/components/Carousel'
+import { Namespaces } from 'src/i18n'
+import { headerWithCancelButton } from 'src/navigator/Headers'
+
+export class VerificationLoadingScreen extends React.Component<WithNamespaces> {
+  static navigationOptions = headerWithCancelButton
+
+  render() {
+    // const { t } = this.props
+    const items: CarouselItem[] = [
+      {
+        text: 'Test 1',
+      },
+      {
+        text: 'Test 2',
+      },
+      {
+        text: 'Test 3',
+      },
+    ]
+    return (
+      <SafeAreaView style={styles.container}>
+        <ScrollView contentContainerStyle={styles.scrollContainer}>
```

Comment:
If there's only the `ScrollView` in this screen (i.e. nothing outside of it), it's nicer on iOS to have the `SafeAreaView` be inside the `ScrollView` so the content doesn't get clipped when overscrolling up or down.